blogposts/universal-code-search-gitlab.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Universal Code Search for GitLab'
+author: Ryan Blunden
+authorUrl: https://twitter.com/ryan_blunden
+publishDate: 2020-03-12T05:59-06:00
+tags: [
+  blog
+]
+slug: universal-code-search-gitlab
+heroImage: /external-logos/gitlab-mark.svg
+published: true
+---
+
+Sourcegraph [Universal Code Search](/universal-code-search) enables developers to explore and better understand all code, faster, with contextual code intelligence.
+
+This screencast (with accompanying audio) shows how to [connect and configure](https://docs.sourcegraph.com/admin/external_service/gitlab) Sourcegraph Universal Code Search for public and private repositories on GitLab.com and GitLab CC/EE instances
+
+In addition to connection configuration, it discusses important factors for determining which repositories Sourcegraph will index, such as excluding archived repositories, as well as repositories based on naming patterns such `deploy-`.
```

Comment:
```suggestion
In addition to configuring the Sourcegraph-GitLab connection, the screencast discusses important factors for setting up your Sourcegraph instance, such as determining which repositories Sourcegraph will index, excluding archived repositories, or including repositories based on naming patterns (e.g.,`deploy-`).
```

Reply:
I incorporated most of these changes, thanks!